__fixtures__/helper.js

```diff
@@ -97,8 +97,12 @@ module.exports = {
               return { data: options.files && options.files }
             }
           },
-          getReviews: async () => {
-            return { data: (options.reviews) ? options.reviews : [] }
+          listReviews: {
+            endpoint: {
+              merge: async () => {
+                return { data: (options.reviews) ? options.reviews : [] }
+              }
+            }
```

Comment:
The arguments to `context.github.paginate` changed - the deprecation warning I got was:

```
Error: .paginate(promise) is deprecated. Use .paginate(endpointOptions) instead.

For example, instead of

    context.github.paginate(context.github.issues.getAll(context.repo())

do

    context.github.paginate(context.github.issues.getAll.endpoint.merge(context.repo())

Note that when using the new syntax, the responses will be mapped to its data only by default.
```